### PR TITLE
Modify GCP playbook for random VPC name support

### PR DESCRIPTION
**Pull request by harshvkarn**
**What this PR does / why we need it**:
- Modify create-vpc.yml to create random vpc instead to hardcoding and
  store the vpc name in a log file
- This allows the playbook to support multiple cluster creation
  simultaneously.
- Modify create-k8s-cluster.yml to read the vpc name from log file and
  use it per se in kops
- Modify delete-vpc.yml to fetch the vpc name from the log file

Signed-off-by: harshvkarn <harshvkarn54@gmail.com>

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: k8s/gcp/k8s-installer/create-k8s-cluster.yml [hunk 1]
```diff
@@ -19,15 +19,22 @@
...
+    - set_fact:
+        vpc: "{{ vpc.stdout }}"
+        
     - name: Creating Bucket
       shell: gsutil mb gs://{{ cluster_name }}/
     - name: Creating the Cluster & InstanceGroup objects in our state store
       shell: |
         export KOPS_FEATURE_FLAGS=AlphaAllowGCE
-        kops create cluster {{ cluster_name }}.k8s.local --kubernetes-version {{ k8s_version }} --zones us-east1-b --state gs://{{ cluster_name }}/ --project {{ project }} --node-count {{ nodes }} --networking kubenet --image "ubuntu-os-cloud/ubuntu-1604-xenial-v20170202" --vpc {{ vpc_name }}
+        kops create cluster {{ cluster_name }}.k8s.local --kubernetes-version {{ k8s_version }} --zones us-east1-b --state gs://{{ cluster_name }}/ --project {{ project }} --node-count {{ nodes }} --networking kubenet --image "ubuntu-os-cloud/ubuntu-1604-xenial-v20170202" --vpc {{ vpc }}
```
Comment (gprasath) [inline k8s/gcp/k8s-installer/create-k8s-cluster.yml]: Are we creating vpc in same zone? And can we have the zone as configurable.

File: k8s/gcp/k8s-installer/create-k8s-cluster.yml [hunk 2]
```diff
@@ -19,15 +19,22 @@
...
     - name: Creating K8s Cluster
       shell: |
         export KOPS_FEATURE_FLAGS=AlphaAllowGCE
```
Comment (gprasath) [inline k8s/gcp/k8s-installer/create-k8s-cluster.yml]: Can we validate if the cluster gets created successfully.

File: k8s/gcp/k8s-installer/delete-vpc.yml [hunk 3]
```diff
@@ -10,11 +10,22 @@
...
+           - set_fact:
+              vpc: "{{ vpc.stdout }}"
            - name: Deleting Routes, if exists
-             shell: gcloud compute routes delete $(gcloud compute routes list --filter="network:{{ vpc_name }}"  --format="get(name)") -q
+             shell: gcloud compute routes delete $(gcloud compute routes list --filter="network:{{ vpc }}"  --format="get(name)") -q
              ignore_errors: yes
            - name: Deleting VPC
-             shell: gcloud compute --project={{ project }} networks delete {{ vpc_name }} -q 
+             shell: gcloud compute --project={{ project }} networks delete {{ vpc }} -q 
+
```
Comment (gprasath) [inline k8s/gcp/k8s-installer/delete-vpc.yml]: Can we check if the network has been deleted successfully.

File: k8s/gcp/k8s-installer/create-k8s-cluster.yml (hunk 1 above)
Comment (harshvkarn) [reply k8s/gcp/k8s-installer/create-k8s-cluster.yml]: Yes, we are making vpc in same zone, i.e. `us-east1-b`, are you suggesting we should be passing zone during runtime?

File: k8s/gcp/k8s-installer/create-k8s-cluster.yml (hunk 2 above)
Comment (harshvkarn) [reply k8s/gcp/k8s-installer/create-k8s-cluster.yml]: Yes, for this we are having a Python script under `utils` that checks for the cluster availability!

File: k8s/gcp/k8s-installer/delete-vpc.yml (hunk 3 above)
Comment (harshvkarn) [reply k8s/gcp/k8s-installer/delete-vpc.yml]: If the VPC is not deleted the above task will fail.

File: k8s/gcp/k8s-installer/create-k8s-cluster.yml (hunk 1 above)
Comment (gprasath) [reply k8s/gcp/k8s-installer/create-k8s-cluster.yml]: yes

Comment (harshvkarn) [reply k8s/gcp/k8s-installer/create-k8s-cluster.yml]: I'll push the commit!

**Pull request merged**